geometry_msgs/Point lt_obstacle_point
geometry_msgs/Point rt_obstacle_point
float32 left_obstacle_distance
float32 right_obstacle_distance
